app/lib/brexit_checker/notifications.yaml

```diff
@@ -23,20 +23,102 @@ notifications:
         - visiting-eu
   # End of example block.
   # Add new notifications below 👇
-  - uuid: b56bc335-3f1f-4b5f-8d58-5bc15cf7a9a5
+  - uuid: b99cc090-d7b0-4eab-93c2-0d6f83df7f87
     type: addition
-    action_id: T129
-    date: '2021-01-15'
+    action_id: T045
+    date: '2021-01-29'
     criteria:
-    - road-passenger-freight
-  - uuid: '08a32a79-4b74-4377-b9e3-7ea90b757069'
+    - all_of:
+      - animal-ex-food
+      - export-to-eu
+  - uuid: ae76f0eb-2096-44b9-8dd3-0f377162e713
     type: addition
-    action_id: T130
-    date: '2021-01-15'
+    action_id: T046
+    date: '2021-01-29'
     criteria:
-    - road-passenger-freight
-  - uuid: a5f51c09-c49c-4143-a0b9-24a22ba772d0
-    type: content_change
+    - all_of:
+      - animal-ex-food
+      - export-to-eu
+  - uuid: e1934fdd-5382-4c38-95be-dbe2752ee533
+    type: addition
+    action_id: T047
+    date: '2021-01-29'
+    criteria:
+    - all_of:
+      - animal-ex-food
+      - export-to-eu
+  - uuid: 5eab1fba-335e-4f2b-b35b-f296b28d3547
+    type: addition
+    action_id: T051
+    date: '2021-01-29'
+  - uuid: 3b55c3e8-51aa-44a7-b003-53474a53bea2
+    type: addition
+    action_id: T088
```

Comment:
The trello card says we are skipping notifications for T088

Reply:
Thanks!